### DISCUSSION TITLE
Thumbnail with 100% zoom crop: possible?

### DISCUSSION THREAD

**Discussion by siovene** [SmileyChris/easy-thumbnails#565] [Q&A]
Apologies if GitHub Issues is not the proper place to ask a question.

I would like to have a thumbnail alias that is a crop of the center of the source image, with a 100% zoom.

For instance, if the source image is 1000x1000, and I define an alias that is 100x100, I would this 100x100 thumbnail to be a 100x100 "window" into the center of the source image, without scaling it.

Is it possible with easy-thumanails?

Thanks!

**Comment by jrief** [comment]
Yes this is possible. However not supported directly by easy-thumbnails. Instead django-filer offers that feature, it's named "[subject location aware cropping](https://django-filer.readthedocs.io/en/latest/installation.html?highlight=subject#subject-location-aware-cropping)".

If you use easy-thumbnails without django-filer, simply pilfer that processor from django-filer.
